agent/rpc/client_grpc.go

```diff
@@ -98,7 +98,6 @@ func (c *client) Next(ctx context.Context, f rpc.Filter) (*rpc.Workflow, error)
 			codes.Aborted,
 			codes.DataLoss,
 			codes.DeadlineExceeded,
-			codes.Internal,
```

Comment:
why this change? 🤔

Reply:
This is explained in the OP (not judging whether it's the right thing to do).